warp10/src/main/java/io/warp10/continuum/gts/GTSEncoder.java

```diff
@@ -1129,10 +1144,14 @@ public synchronized void merge(GTSEncoder encoder) throws IOException {
       this.lastBDValue = encoder.lastBDValue;
       this.lastDoubleValue = encoder.lastDoubleValue;
       this.lastStringValue = encoder.lastStringValue;
+
+      // Disable delta encoding, this allows us to have a wider use of the fast path
+      safeDelta();
```

Comment:
Instead copy last loc/elev/values according to valid* booleans.

Reply:
Added the copying of the valid* flags, but left the noDeltaMetaTimestamp forced to true by leaving the call to safeDelta as a first step